jaeger/charts/linkerd-jaeger/values.yaml

```diff
@@ -32,6 +32,9 @@ enablePodAntiAffinity: false
 # for more information
 #nodeAffinity:
 
+# -- adding an option to disble namespace-metadata jobs
+skipNamespaceMetadataJobs: false
```

Comment:
```suggestion
# -- Creates a Job that adds necessary metadata to the extension's namespace
# during install; disable if lack of privileges require doing this manually
createNamespaceMetadataJobs: true
```

Reply:
I have made the suggested changes and regenerated the helm values documentation in the charts README.md files.